### ISSUE TITLE
learning Rate

### ISSUE THREAD

**Issue by zhou-frank** [fizyr/keras-retinanet#194]
Why your model's lr set so low(0.00001)? On the contrary，the paper set it to 0.01

**Comment by yhenon** [comment]
The settings in this repo are for a minibatch size of 1, with Adam optimiser. The settings in the paper are for a minibatch size of 16, with SGD optimiser.

Generally larger minibatches are more stable and should use larger learning rates. The choice of optimiser has an effect too, but Adam is generally faster. Finally, you shouldnt compare LRs across different optimisers.

**Comment by chaitons** [comment]
Also, the training is much more stable if you add regularization.

**Comment by hgaiser** [comment]
Do you have experiments you can share with us that show your case?
